parm/config/gfs/config.wave

```diff
@@ -54,43 +54,44 @@ case "${waveGRD}" in
     #Grid used for P8
     export waveinterpGRD=''
     export wavepostGRD='gwes_30m'
+    export waveuoutpGRD=${waveGRD}
     ;;
   "mx025")
     #Grid used for HR1 (tripolar 1/4 deg)
     export waveinterpGRD='reg025'
-    export wavepostGRD=' '
+    export wavepostGRD=''
+    export waveuoutpGRD=${waveGRD}
     ;;
   "glo_025") 
     #GEFSv13 regular lat/lon 1/4 deg grid 
     export waveinterpGRD=''
     export wavepostGRD='glo_025'
+    export waveuoutpGRD=${waveGRD}
     ;;
   "glo_200")
     #Global regular lat/lon 2deg deg grid 
     export waveinterpGRD=''
     export wavepostGRD='glo_200'
+    export waveuoutpGRD=${waveGRD}
     ;;
   "glo_500")
     #Global regular lat/lon 5deg deg grid 
     export waveinterpGRD=''
     export wavepostGRD='glo_500'
+    export waveuoutpGRD=${waveGRD}
     ;;
   *)
     echo "No grid specific wave config values"  
+    export waveinterpGRD=${waveinterpGRD:-'glo_15mxt at_10m ep_10m wc_10m glo_30m'} # Grids that need to be interpolated from native 
```

Comment:
If you choose a waveGRD that is not one of the specified values above. This would be more of a case where you could try to use a grid that's not one of the set-grids here but you have fewer places to change things.  I guess this could be removed completely instead of just moved here.

Reply:
I guess the more appropriate question is, is there a use case for this `*`?  Should we instead abort saying this `waveGRD` is not supported and ask the user to choose one from the above available options?

Reply:
My opinion is no, but if that's the desired path forward I can make those changes.

Reply:
Apologies, I mispoke here.  My opinion is no we should not exit.   But since that's what you want I'll  go ahead and make the change so that this bug fix can be made available as soon as possible.

Reply:
This update has been done as requested.